### PR TITLE
added counter steps functionality

### PR DESCRIPTION
**Pull request by soniaigartua**
added counter steps functionality

Evidence:

### CODE REVIEW

File: app/src/main/res/layout/activity_steps_counter.xml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            android:textColor="@color/black_color"
+            android:textSize="@dimen/font_size_weather_desc" />
+    </LinearLayout>
+
+
+    <LinearLayout
+        android:id="@+id/weather_button"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="20dp"
```
[the remark this answers is not in the document]
Comment (soniaigartua) [reply app/src/main/res/layout/activity_steps_counter.xml]: done!

File: app/src/main/res/layout/activity_steps_counter.xml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+        <TextView
+            android:id="@+id/counter_tittle"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_horizontal"
+            android:text="Total Steps"
+            android:textAllCaps="true"
+            android:textColor="@color/black_color"
+            android:textSize="25sp" />
```
[the remark this answers is not in the document]
Comment (soniaigartua) [reply app/src/main/res/layout/activity_steps_counter.xml]: DONE!

File: app/src/main/res/layout/activity_steps_counter.xml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+        <TextView
+            android:id="@+id/number_steps"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="0"
+            android:textColor="@color/black_color"
+            android:textSize="70sp" />
```
[the remark this answers is not in the document]
Comment (soniaigartua) [reply app/src/main/res/layout/activity_steps_counter.xml]: DONE!

File: app/src/main/res/layout/activity_steps_counter.xml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+        <Button
+            android:id="@+id/btn_return"
+            style="@style/ButtonStyle"
+            android:layout_marginTop="30dp"
```
[the remark this answers is not in the document]
Comment (soniaigartua) [reply app/src/main/res/layout/activity_steps_counter.xml]: done

File: app/src/main/res/layout/activity_steps_counter.xml [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+            android:text="Total Steps"
+            android:textAllCaps="true"
+            android:textColor="@color/black_color"
+            android:textSize="25sp" />
+
+        <TextView
+            android:id="@+id/number_steps"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="0"
```
[the remark this answers is not in the document]
Comment (soniaigartua) [reply app/src/main/res/layout/activity_steps_counter.xml]: done

File: app/src/main/java/com/example/pps_tudai/mvp/presenter/StepsCounterPresenter.java [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+import static com.example.pps_tudai.utils.StringUtils.TRUE;
+
+public class StepsCounterPresenter implements SensorEventListener {
+
+    private static final int ONE = 1;
+    private final StepsCounterView counterView;
+    private final StepsCounterModel counterModel;
+    private User user;
+    SensorManager sensorManager;
+    Sensor sensor;
```
Comment (nicolbidone) [inline app/src/main/java/com/example/pps_tudai/mvp/presenter/StepsCounterPresenter.java]: sensorManager & sensor should be private

Comment (soniaigartua) [reply app/src/main/java/com/example/pps_tudai/mvp/presenter/StepsCounterPresenter.java]: fixed it

**Pull request merged**